### PR TITLE
Support Unique operator in ONNX

### PR DESCRIPTION
**Pull request by hariharans29**
Copy of #2002

### CODE REVIEW

File: onnx/defs/tensor/defs.cc [hunk 1]
```diff
@@ -1822,4 +1822,65 @@ ONNX_OPERATOR_SET_SCHEMA(
...
+  output_idx = [0, 1, 1, 2, 3, 2]
+  output_counts = [1, 2, 2, 1]
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    Unique,
+    11,
+    OpSchema()
+        .SetDoc(Unique_ver11_doc)
+        .Input(0, "x", "A 1-D input tensor that is to be processed.", "T")
```
Comment (liqunfu) [inline onnx/defs/tensor/defs.cc]: Please support for general n-D tensors as detailed here:
https://pytorch.org/docs/stable/torch.html?highlight=unique#torch.unique

**Comment by hariharans29** [comment]
CC: @pranavsharma @jeffsaremi @jiafatom @faxu - Closing this PR as per offline discussion since there is no immediate business requirement. This PR can be re-opened and iterated upon later if need be.

**Comment by liqunfu** [comment]
@hariharans29, there is a pytorch-translate model that need unique op with n-D tensors. Please re-open this PR and make it support n-D cases. Thanks.

**Comment by hariharans29** [comment]
Hi @liqunfu  - Please feel free to iterate on this if you have immediate need.

**Pull request closed without merging**